Sources/SplashImageGen/Extensions/CommandLine+Options.swift

```diff
@@ -46,6 +48,16 @@ extension CommandLine {
 
         return Font(path: path, size: size)
     }
+    
+    private static func resolveTheme(from defaults: UserDefaults) -> ThemeName {
+        guard let providedName = defaults.string(forKey: "t") else {
```

Comment:
`let themeName = ThemeName(rawValue: providedName)` could be added to this guard. To only have two returns instead of three.